bugsnag-android-core/src/test/java/com/bugsnag/android/ConfigurationFacadeTest.java

```diff
@@ -0,0 +1,426 @@
+package com.bugsnag.android;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import androidx.annotation.NonNull;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+@SuppressWarnings("ConstantConditions") // suppress warning about making redundant null checks
+public class ConfigurationFacadeTest {
```

Comment:
Tests missing for removeOnError, removeOnBreadcrumb and removeOnSession.
More tediously, there's no coverage for the getters - but I can see why you've put assertions on the values from impl and I think we should keep them.  Covering the getters may feel like nugatory effort, but the benefit for us is that 100% coverage means we never need to look closer to check what is/isn't covered (and potentially we can also put a build check on it).  Having said that I suspect Alex already has some tests that would cover them.

Reply:
Added tests for the missing methods. Assuming no change is required on how the getters are tested since we have some test coverage on them.